### PR TITLE
Reduced linearsolver option

### PR DESCRIPTION
**Pull request by hnil**
- added possibility to use reduced tolerance for linear solver

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

File: opm/simulators/linalg/ISTLSolverEbos.hpp [hunk 1]
```diff
@@ -435,6 +435,10 @@ std::unique_ptr<Matrix> blockJacobiAdjacency(const Grid& grid,
             iterations_ = result.iterations;
             converged_ = result.converged;
             if(result.reduction < parameters_.relaxed_linear_solver_reduction_){
+                std::stringstream ss;
+                ss<< "Full linear solver tolerance not achived. The reduction is:" << result.reduction
+                  << " after " << result.iterations << " iteraions ";
```
Comment (akva2) [inline opm/simulators/linalg/ISTLSolverEbos.hpp:440]: iterations

Comment (hnil) [reply opm/simulators/linalg/ISTLSolverEbos.hpp:440]: The intention is just to accept a linear solve which after max iterations did not converged, but the solution is not that bad. This make it possible to set the max iteration quite low without triggering time step cuts due to none convergence and at the same time not continue if linear solves is clearly going in the wrong direction.

Comment (akva2) [reply opm/simulators/linalg/ISTLSolverEbos.hpp:440]: sure i got that. but we still don't want the typo ;)

Comment (hnil) [reply opm/simulators/linalg/ISTLSolverEbos.hpp]: done

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**